vendor/github.com/redhat-cne/sdk-go/pkg/subscriber/subscriber.go

```diff
@@ -84,8 +84,8 @@ func (s *Subscriber) FailedCount() int {
 // String returns a pretty-printed representation of the Event.
 func (s *Subscriber) String() string {
 	b := strings.Builder{}
-	b.WriteString("  EndPointURI: " + s.GetEndPointURI() + "\n")
-	b.WriteString("  ID: " + s.GetClientID().String() + "\n")
```

Comment:
Same here

Reply:
no. tested with old version. The subscription was successful.